src/main/java/uk/gov/cshr/civilservant/service/OrganisationalUnitService.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.cshr.civilservant.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.cshr.civilservant.domain.OrganisationalUnit;
+import uk.gov.cshr.civilservant.repository.OrganisationalUnitRepository;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@Service
+@Transactional
+public class OrganisationalUnitService {
+
+    private OrganisationalUnitRepository organisationalUnitRepository;
+
+    @Autowired
+    public OrganisationalUnitService(OrganisationalUnitRepository organisationalUnitRepository) {
+        this.organisationalUnitRepository = organisationalUnitRepository;
+    }
+
+    public ArrayList<OrganisationalUnit> getParentOrganisationalUnits() {
+        ArrayList<OrganisationalUnit> organisationalUnitArrayList = new ArrayList<>();
+
+        Iterable<OrganisationalUnit> organisationalUnits = organisationalUnitRepository.findAll();
+
+        organisationalUnits.forEach(organisationalUnit -> {
+            OrganisationalUnit currentNode = organisationalUnit;
+            if (!currentNode.hasParent()) {
+                organisationalUnitArrayList.add(currentNode);
+            }
+        });
+
```

Comment:
I would suggest changing the OrganisationalUnitRepository from A CrduRepository to a JpaRepository then findAll will return a List. You could then write this method as...
```
    public List<OrganisationalUnit> getParentOrganisationalUnits() {

        return organisationalUnitRepository.findAll().stream()
                .filter(o -> !o.hasParent())
                .collect(Collectors.toList());
    }
```